polylogue/ui/facade.py

```
@@ -72,28 +139,78 @@ def summary(self, title: str, lines: Iterable[str]) -> None:
+        if response is not None:
+            if response.get("use_default"):
+                return options[0] if options else None
+            if "value" in response and response["value"] in options:
+                return response["value"]
+            if "index" in response:
+                try:
+                    idx = int(response["index"])  # type: ignore[arg-type]
+                    if 0 <= idx < len(options):
+                        return options[idx]
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)